src/test/unit/java/org/xpdojo/bank/MoneyTest.java

```diff
@@ -9,43 +11,45 @@
 class MoneyTest {
 
 	@Test
-	void addingMonies() {
+	void addingAnAmountShouldGiveTheSumOfTheTwoAmounts() {
 		assertThat(amountOf(10).plus(amountOf(14)), is(amountOf(24)));
 	}
 
 	@Test
-	void addingNegativeMonies() {
+	void addingANegativeAmountShouldSubtractTheEquivalentPositiveAmount() {
 		assertThat(amountOf(10).plus(amountOf(-14)), is(amountOf(-4)));
 	}
 
 	@Test
-	void takingMoniesAway() {
+	void subtractingAnAmountShouldGiveTheSubtractedSum() {
 		assertThat(amountOf(14).minus(amountOf(10)), is(amountOf(4)));
 	}
 
 	@Test
-	void takingMoniesAwayWhenSubtractionIsGreaterThanInitialValue() {
+	void subtractingMoreThanTheCurrentAmountShouldGiveANegativeResult() {
 		assertThat(amountOf(10).minus(amountOf(14)), is(amountOf(-4)));
 	}
 
 	@Test
-	void takingNegativeMoniesAway() {
+	void subtractingANegativeAmountShouldAddTheEquivalentPositiveAmount() {
 		assertThat(amountOf(10).minus(amountOf(-14)), is(amountOf(24)));
 	}
 
-	@Test
-	void lessThan() {
-		assertThat(amountOf(0).isLessThan(amountOf(0)), is(false));
-		assertThat(amountOf(10).isLessThan(amountOf(10)), is(false));
-		assertThat(amountOf(10).isLessThan(amountOf(11)), is(true));
-		assertThat(amountOf(11).isLessThan(amountOf(10)), is(false));
-		assertThat(amountOf(-10).isLessThan(amountOf(10)), is(true));
-		assertThat(amountOf(10).isLessThan(amountOf(-10)), is(false));
+	@ParameterizedTest(name = "{0} is less than {1} should be {2}")
```

Comment:
lol, I don't think I like JUnit 5 🤓

Reply:
I know right?! I'm 50/50 which is more readable. The only thing that swings it this way for me is that it results in one assertion per test and doesn't bomb out at the first failure.